scripts/docker-compose.yml

```diff
@@ -79,38 +80,69 @@ services:
       POSTGRES_USER: insights
       POSTGRES_DB: insights
     ports:
-        - "15433:5432"
-  insights-inventory:
+      - "15433:5432"
+
+  insights-inventory-mq: &inventory
     image: quay.io/cloudservices/insights-inventory:latest
     restart: always
-    ports:
-      - 8081:8081
+    command: "make upgrade_db run_inv_mq_service"
     environment:
-      - APP_FILE=run_gunicorn.py
       - APP_NAME=inventory
       - PATH_PREFIX=api
       - INVENTORY_DB_USER=insights
       - INVENTORY_DB_PASS=insights
       - INVENTORY_DB_HOST=db-host-inventory
       - INVENTORY_DB_NAME=insights
       - KAFKA_BOOTSTRAP_SERVERS=kafka:29092
+      - KAFKA_TOPIC=platform.system-profile
+      - KAFKA_GROUP=inventory
       - PAYLOAD_TRACKER_ENABLED=false
       - LISTEN_PORT=8081
     depends_on:
        - kafka
        - db-host-inventory
        - puptoo
-       - zookeeper
+
+  insights-inventory-web:
+    <<: *inventory
```

Comment:
I didn't get the idea of running inventory container two times, I can also see that migration is also running twice.